sns/cm/dtrebalance/cm.c

```diff
@@ -149,39 +149,39 @@ static void dtrebalance_cm_stop(struct m0_cm *cm)
 	M0_LEAVE();
 }
 
-/**
- * Returns 0 iff the copy machine has enough space to receive all
- * the copy packets from the given relevant group "id".
- * Calculates required number of buffers from incoming buffer pool
- * struct m0_sns_cm::sc_ibp corresponding to all the incoming copy packets
- * and checks if the incoming buffer pool has enough free buffers to
- * receive all the remote units corresponding to a parity group.
+/*
+ * Checks if the incoming buffer pool has enough free buffers to
+ * receive more copy packets.
+ * Returns 0 iff the copy machine has space.
  */
 static int dtrebalance_cm_get_space_for(struct m0_cm *cm,
```

Comment:
This function might be useful to check number of buffer required for an aggregation group. Better to introduce some other function for this purpose

Reply:
Introducing a new function will require adding logic in the caller to selectively call the new function for direct rebalance. That may require adding conditional execution and special handling for direct rebalance.